### ISSUE TITLE
Is there an API with offline navigation?

### ISSUE THREAD

**Issue by FrancisFYK** [organicmaps/api-android#6]
We would like to integrate offline maps and offline navigation. Can you support this?

**Comment by sandy99405** [comment]
Yes I would like to support that. Could you please provide the specifications you want to integrate ?
